content/docs/guides/koyeb.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Use Neon with Koyeb
+enableTableOfContents: true
+---
+
+[Koyeb](https://www.koyeb.com/) is a developer-friendly, serverless platform designed to easily deploy reliable and scalable applications globally. Koyeb offers native autoscaling, automatic HTTPS (SSL), auto-healing, and global load-balancing across their edge network with zero configuration.
+
+This guide describes how connect a Neon PostgreSQL database to an application deployed with Koyeb. To follow the instructions in this guide, you require:
+
+- A [Koyeb account](https://app.koyeb.com/) to deploy the application. Alternatively, you can install the [Koyeb CLI](https://www.koyeb.com/docs/quickstart/koyeb-cli) if you prefer to deploy the application from your terminal.
+- A [Neon account](https://console.neon.tech/) to deploy the PostgreSQL database.
+
+The example application connects to your Neon PostgreSQL database using [Prisma](https://www.prisma.io/) as an ORM. Prisma synchronizes the database schema with the Prisma schema included with the application and seeds the database.
+
+## Create a Neon project
+
+1. Navigate to the [Neon Console](https://console.neon.tech/).
+1. Select **Create a project**.
+1. Enter a name for the project (`neon-koyeb`, for example), and select a PostgreSQL version and region.
+1. Click **Create project**.
+
+A dialog pops up with your Neon connection string, which appears similar to the following:
+
+```bash
+postgres://<user>:<password>@<host>:<port>/neondb
+```
+
+Store this value in a safe place. It is required later. The connection string specifies `neondb` as the database. This is the default database created with each Neon project. You will use this database with the example application.
+
+## Deploy the application on Koyeb
+
+You can deploy on Koyeb using the control panel or the Koyeb CLI.
+
+### From the Koyeb control panel
+
+To deploy the application from the Koyeb [control panel](https://app.koyeb.com/), follow these steps:
+
+1. Select **Create App**.
+1. Select GitHub as the deployment method.
+1. Enter `https://github.com/koyeb/example-express-prisma` in the **Public GitHub repository** field.
+1. Keep `example-express-prisma` as the name and `main` as the branch.
+1. In **Build and deployment settings**, enable the **Override** setting and add the following **Build command**: `npm run postgres:init`
+1. Select the region closest to your Neon database.
+1. Under **Advanced** > **Environment variables**, add a `DATABASE_URL` environment variable to enable the application to connect to your Neon PostgreSQL database. Set the value to the Neon connection string provided to you when you created the Neon project.
+1. Enter a name for your app. For example, `express-neon`
+1. Click **Deploy**.
+
+Koyeb builds the application. Once the build has finished, you can access your application running on Koyeb by clicking the URL ending with `.koyeb.app`.
```

Comment:
Once the build has finished, you can access your application running on Koyeb by clicking the URL ending with .koyeb.app.

->

Once the build and deployment have finished you can access your application running on Koyeb by clicking the URL ending with `.koyeb.app`.

Reply:
Thank you. I will address this in another PR.